charts/gardener-extension-validator-vsphere/charts/runtime/templates/deployment.yaml

```diff
@@ -46,6 +46,9 @@ spec:
         {{- if .Values.global.kubeconfig }}
         - --kubeconfig=/etc/gardener-extension-validator-vsphere/kubeconfig/kubeconfig
         {{- end }}
+        {{- if .Values.metricsPort }}
```

Comment:
I'm guessing you meant to say `{{- if .Values.global.metricsPort }}` here.

Reply:
Thank you!